styles.css

```diff
@@ -0,0 +1,3 @@
+* {
+  display: block;
+}
```

Comment:
- [ ] I would suggest you remove this code as it is giving problems when I open in the browser.
### Error
![sssss](https://user-images.githubusercontent.com/81727988/182420393-8d86d13f-858a-4361-83b1-c98e26131176.png)